src/transformers/onnx/features.py

```diff
@@ -202,6 +202,16 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls="models.data2vec.Data2VecTextOnnxConfig",
         ),
+        "deberta-v2": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
```

Comment:
I'm not sure if there is `causal-lm` as a supported feature for Debertav2.

Reply:
To confirm `DebertaV2ForCausalLM` is not implemented, shall remove `causal-lm`.

Reply:
done